packages/frontend/components/Borrow/CollateralDropdown.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import { MenuItem, Typography } from "@mui/material"
+
+import { chains } from "../../machines/auth.machine"
+import Image from "next/image"
+
+type Chain = typeof chains[0]
+
+type CollateralDropdownProps = {
+  chains: Chain[]
+}
+
+export default function CollateralDropdown(props: CollateralDropdownProps) {
```

Comment:
I'm not sure about the cutting. I had the same concerns about the `settings` menu you created.

1. I'm not sure a component is neededbecause it contains no logic (presentational component) and will never be reused

2. A component is a reusable piece of UI. To me it make no sense to have only a dropdown without the button to trigger it 🤔 

3. It's not a Collateral Dropdown, but rather Chain Dropdown

Reply:
Resolved by talking directly